src/pip/_internal/cli/base_command.py

```diff
@@ -6,6 +6,7 @@
 import os
 import sys
 import traceback
+from functools import wraps
```

Comment:
Nit: `import functools` seems to fit existing style beter.

Reply:
Agreed - c75ad51dc.